src/cmd/arc.rs

```diff
@@ -42,10 +42,10 @@ pub fn run(args: &Args) -> Result<()> {
 pub struct ArgsGuestKernelUprev {
     /// target cros repo dir
     #[argh(option)]
-    repo: Option<String>,
+    cros: Option<String>,
 }
 fn run_guest_kernel_uprev(args: &ArgsGuestKernelUprev) -> Result<()> {
-    let chroot = Chroot::new(&get_repo_dir(&args.repo)?)?;
+    let chroot = Chroot::new(&get_repo_dir(&args.cros)?)?;
```

Comment:
Can you also rename `get_repo_dir` to `get_cros_dir`?

Reply:
Sure, I fixed it!